Server/RFIDNamespace.py

```diff
@@ -17,11 +22,39 @@ class RFID(Resource):
     @api.response(200, "success")
     @api.response(404, "Unknown Card")
     def get(self, card_id):
-        # TODO: Actually check a database for the known cards.
-        if card_id != "8666529cc":
+        user = User.query.filter_by(card_id = card_id).first()
+        if not user:
             return UNKNOWN_CARD_RESPONSE
         else:
             return "Welcome back!"
 
-
+# name, email are required for new users. Ability can be passed multiple times.
+# Only adds abilities; does not remove them.
+@RFID_namespace.route("/update/<string:card_id>/")
+@RFID_namespace.doc(description ="Update a user's information")
+class RFID(Resource):
+    @api.response(200, "User updated")
+    @api.response(500, "User update failed")
+    def get(self, card_id):
```

Comment:
Doesn't it conceptually make more sense to add new users with a post? As far as I've understood it, a get request shouldn't change any data.

Reply:
Good call